### PR TITLE
vimdiff, vimtutor: add German translation

### PR DESCRIPTION
**Pull request by spicyyboi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/vimtutor.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# vimtutor
+
+> Vim-Tutor zum Erlernen grundlegender Vim-Befehle.
+> Weitere Informationen: <https://manned.org/vimtutor>.
+
+- Öffne den Vim-Tutor in englischer|französischer|deutscher|... Sprache:
+
+`vimtutor {{en|fr|de|...}}`
```
Comment (kbdharun) [inline pages.de/common/vimtutor.md:8]:
```suggestion
`vimtutor {{Sprache}}`
```

Comment (pixelcmtd) [inline pages.de/common/vimtutor.md:8]:
```suggestion
- Öffne den Vim-Tutor in der gegebenen Sprache (de, en, fr, ...):

`vimtutor {{sprache}}`
```

Comment (spicyyboi) [reply pages.de/common/vimtutor.md:8]: This sounds reasonable as vimtutor only accepts the language code and not the language itself (at least on my machine).

**Review by kbdharun — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @spicyyboi

**Pull request merged**